AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-only-procedure.rst

```diff
@@ -0,0 +1,54 @@
+############################################
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Overview:
+Overview
+========
+This procedure will acquire BIAS and DARK calibrations images. After the entire script completion, the observer can decide whether to vent, proceed to on-sky or shutdown.
+This page assumes the reader is familiar with the content explained in the Observatory section: :ref:`Introduction to the combined calibrations generation procedure <Introduction-Combined-Calibrations-Generation-index>` 
+
+.. warning::
+  This procedure does not involves telescope motion with AuxTel. Announce that you are about to run the calibrations without telescope or dome movement on the *#summit-announce* and *#summit-auxtel* channels before you run the following steps.
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Post-Conditions:
+Post-Condition
+==============
+- Individual BIAS and DARK calibration images will be taken and verified using the `cp_verify`_ framework.
+- If ``generate_calibrations`` is ``True``, a (daily) combined calibration image per detector will be certified in a `butler`_ ``CALIBRATION`` `collection`_.
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Procedure-Steps:
+Procedure Steps
+===============
+#. Calibrations images should only be taken if the LATISS WREB temperature is under temperature control. Check the LATISS WREB temperatures under the `AuxTel (LATISS) Temperatures and Pressures dashboard`_ in Chronograf. WREB temperatures are visible in the lower middle panel labeled *WREB On Board*. If the *mean_temp2* (top blue line) is between 26-29 degress C, the temperature is suitable for taking calibrations. During the daytime, the fan on the WREB board may not be sufficient to cool the WREB down to these temperatures, so during warmer months you may have to wait until later in the day or early in the morning for it to reach the desired temperature. If the temperature is too high, do not proceed to the next steps. 
+#. Enable ATCS and LATISS using the standard scripts :file:`enable_atcs.py` and :file:`enable_latiss.py` with no configuration. 
+#. Enable the ``OCPS:2`` scheduler. Use the standard script :file:`auxtel/scheduler/enable.py` with the configuration required for the run and available in the corresponding night log. 
+#. Run the script :file:`auxtel/make_latiss_calibrations.py` on the ATQueue  with the following configuration:
+
+.. code-block:: text
+  :caption: :file:`auxtel/make_latiss_calibrations.py`
+  
+    script_mode: BIAS_DARK
+
+The script takes the default number of BIAS (20) and DARK (22 with different exposure times  5, 15 and 30 seconds) calibration images. The process will take about 15 minutes to complete.
+
+Once the script finishes, announce that the calibrations are complete on *#summit-announce* and *#summit-auxtel* channels. Use the :file:`standby_latiss.py` standard script to leave LATISS in ``STANDBY`` state.
```

Comment:
This would be step number 5. In order for the numbering to continue you must indent every part of each step (A vertical line will appear showing you that's inside the bullet point)

Reply:
Thank you Ioana. All your comments were addressed.